xml/obs_ag_installation_and_configuration.xml

```diff
@@ -249,6 +249,19 @@ systemctl start obswarden.service</screen>
      not do this on a system connected to an untrusted network or make sure to
      block the ports with a firewall.</para>
    </warning>
+   <sect3 xml:id="_cloud_upload_setup">
+    <title>Cloud Upload Setup</title>
+    <para>In order to setup the Cloud Upload feature you will need to configure the tools required per each cloud provider.</para>
+    <para>
+      Right now we only support the AWS Amazon Cloud (<link xlink:href="https://aws.amazon.com"/>) as provider. For uploading images in AWS two tools are needed the "aws CLI" and the "ec2uploadimg".
```

Comment:
Is it necessary to mention which tools we use in the backend? ->

Reply:
If the admin have to configure those... I guess so

Reply:
The tools get automatically installed by the package so I wouldn't mention it here. I would just mention the config files.

Reply:
Just mentioning the config files and pointing to examples and a page that explains how to config the tool  is not 99% of the same?

Reply:
I'm just wondering if you state the tools explicitly the Admin might wonder if it is necessary to actually install the tools, no?

Reply:
Ok, then maybe saying that the admin don't need to install the tools will be ok?

Reply:
Done...